Sources/SocksCore/SocketOptions+Deprecated.swift

```diff
@@ -0,0 +1,100 @@
+//
```

Comment:
Not sure what you mean by that. Almost all files in SocksCore have headers. Should I just omit any header in new files?